src/components/Navbar/Navbar.spec.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { render, screen } from '@testing-library/react';
+import Navbar from './Navbar.component';
+
+
+describe('Navbar Component', () => {
+    
+    beforeEach(() => {
+        render(<Navbar />);
+    });
+
+    it('should match snapshot', () => {
```

Comment:
cool, I like the thing that you used snapshots as a initial test for your components